tests/test_live_read_write.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+"""Test gs-wrap read write live."""
+
+# pylint: disable=missing-docstring
+# pylint: disable=too-many-lines
+# pylint: disable=protected-access
+# pylint: disable=expression-not-assigned
+
+import unittest
+import uuid
+
+import temppathlib
+
+import gswrap
+import tests.common
+
+
+class TestReadWrite(unittest.TestCase):
+    def setUp(self):
+        self.client = gswrap.Client(bucket_name=tests.common.TEST_GCS_BUCKET)
+        self.bucket_prefix = str(uuid.uuid4())
+
+    def tearDown(self):
+        pass
+
+    def test_read_bytes(self):
+        with temppathlib.NamedTemporaryFile() as file:
+            file.path.write_bytes(tests.common.GCS_FILE_CONTENT.encode('utf-8'))
+            tests.common.call_gsutil_cp(
+                src=file.path.as_posix(),
+                dst="gs://{}/{}/file".format(tests.common.TEST_GCS_BUCKET,
+                                             self.bucket_prefix),
+                recursive=False)
+
+            content = self.client.read_bytes(url="gs://{}/{}/file".format(
```

Comment:
in case of a crash here you would not cleanup the bucket. You need to call something in `tearDown` or in try/finally block

Reply:
Good catch!